core/src/main/java/org/mule/runtime/core/internal/processor/strategy/reactor/builder/ComponentProcessingStrategyReactiveProcessorBuilder.java

```diff
@@ -137,11 +138,12 @@ public ReactiveProcessor build() {
     Optional<ProfilingDataProducer<ComponentProcessingStrategyProfilingEventContext>> psFlowMessagePassingDataProducer =
         dataProducerFromProfilingService(PS_FLOW_MESSAGE_PASSING);
 
-    // location
+    // Component data
     ComponentLocation location = getLocation(processor);
```

Comment:
Is something missing here? getComponentData or something like that?

Reply:
I don't think so.